react-live.css

```diff
@@ -92,3 +92,9 @@
 .token.deleted {
   color: red;
 }
+
+.tabGuarded{
```

Comment:
The problem is that now we are directly focusing the editor so it will have an outline the entire duration of the editing.... or that's what I was seeing.  I'll mess with the overflow you mentioned.